packages/airnode-protocol/scripts/run-hardhat-tests.ts

```diff
@@ -0,0 +1,50 @@
+// NOTE: Hardhat tests are terribly slow when there are large number of tests present
```

Comment:
Where is this script called from? I can't find anything that calls `run-hardhat-test.ts`

Reply:
I've actually accidentally removed the `test` command (probably during cleanup rebase) :pleading_face: 

I'll add it back - hopefully the test metric I did was done when the `test` command was present :)